src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -828,6 +828,54 @@ public static BinaryOperandsKind GetBinaryOperandsKind(BinaryOperationKind kind)
         }
     }
 
+    public interface IConversion
+    {
+        /// <summary>
+        /// Returns true if the conversion exists, either as an implicit or explicit conversion.
+        /// </summary>
+        bool Exists { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an identity conversion.
+        /// </summary>
+        bool IsIdentity { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit ('widening' in VB) conversion.
+        /// </summary>
+        bool IsImplicit { get; }
+
+        /// <summary>
+        /// True if and only if the conversion is an explicit ('narrowing' in VB) conversion.
+        /// </summary>
+        bool IsExplicit { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit numeric conversion or explicit numeric conversion. 
+        /// </summary>
+        bool IsNumeric { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit nullable conversion or explicit nullable conversion.
+        /// </summary>
+        bool IsNullable { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit reference conversion or explicit reference conversion.
+        /// </summary>
+        bool IsReference { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit user-defined conversion or explicit user-defined conversion.
+        /// </summary>
+        bool IsUserDefined { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit default ('null' in C#, 'nothing' in VB) literal conversion.
+        /// </summary>
+        bool IsDefault { get; }
```

Comment:
I was trying to pick a good name here.  VB calls it  'IsDefault' and associates it with the 'Nothing' literal.  I was trying ot think of a good name to represent nothing in VB and null in C# and i went with IsDefault since we already call it that elsewhere.

I'm happy to change this to a better name (or we could just remove this for now).   Do you have a recommendation?